app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1 @@
+@import "bootstrap/scss/bootstrap";
```

Comment:
verify if required here

Reply:
bootstrap's css does not load without it